### DISCUSSION TITLE
Can I shutdown and later restart lustre that uses local ssd?

### DISCUSSION THREAD

**Discussion by hpc-admin** [GoogleCloudPlatform/cluster-toolkit#1164] [Ideas and feature requests]
I want to try local SSDs for my lustre OST and MDT drives. However, when I shutdown the nodes and later restart them, no local SSDs are attached. I don't need data to persist between shutdowns but it would be nice if fresh local SSDs are attached to the VMs when they are started. Is this possible?

**Comment by cboneti** [comment]
Here I am making the assumption you are using `community/modules/file-system/DDN-EXAScaler/`.  Please correct me if this is not the case.

As you mention, the data in the local SSDs does not persist [if you shut down the guest operating system and force the instance to stop](https://cloud.google.com/compute/docs/disks/local-ssd#data_persistence). 

Effectively this would mean recreating the file system on reboot and I am wondering would would it not be simpler to, instead, just destroy and recreate the system (rather than stopping the machine). 

Could you please clarify?

**Reply by hpc-admin** [reply]
Yes, I am using the DDN-exascaler community module. I can destroy and recreate the machine but then I believe I need to destroy and recreate the slurm cluster that mounts the lustre filesystem as the IP address of the MGS node will have changed. I find this a bit tedious so I was wondering if there was a simpler way. If its not possible I am ok with destroying/recreating.

**Reply by cboneti** [reply]
This makes sense. I can try to propose something that may achieve that goal. 

Are you trying to delete the lustre file system without deleting the cluster? Do you keep one blueprint for the file systems and another for the cluster, or are you using multiple deployment groups?

**Reply by cboneti** [reply]
Hi, thanks for the patience here.

Just to close the loop, I will submit the feature request to DDN but I am not sure how that will be prioritized since it might come with some implementation complexity. 

In addition, what I can do here is try to understand a little more about your use case and figure out if we can propose an alternative.  

So far, our understanding was that destroying the full deployment (slurm and luster) would not be any harder.  User configurations such as home file systems and applications can be kept in another, smaller or cheaper, file system that is deployed in a previous deployment group or even in a GCS bucket. Does that make sense?

**Reply by hpc-admin** [reply]
Yes using a small, cheap filesystem for the home directory works, thank you.
